apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -50,6 +50,26 @@ model DiscordUser {
   discord_id String @unique
   user_id Int
   user User @relation(fields: [user_id], references: [id])
+
+  // notifications
+  // enable notifications
+  enable_notifications Boolean @default(false)
+
+  // submit contributions
+  enable_submit_reminder_period Boolean @default(false)
```

Comment:
These look good, on the the enable columns like this it may be clearer to remove the _period suffix. So instead of the above it should be enable_submit_reminder